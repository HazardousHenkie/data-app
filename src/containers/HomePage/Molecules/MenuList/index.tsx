import React from 'react'

import List from '@material-ui/core/List'
import ListItem from '@material-ui/core/ListItem'
import ListItemText from '@material-ui/core/ListItemText'

import ThemeSwitcher from 'components/Atoms/ThemeSwitcher'
import LanguageSwitcher from 'components/Molecules/LanguageSwitcher'

const MenuList: React.FC = () => {
    return (
        <List component="nav" aria-label="main menu">
            <ListItem>
                <ListItemText primary="home" />
            </ListItem>

            <ListItem>
                <LanguageSwitcher />
            </ListItem>
            <ListItem>
                <ThemeSwitcher />
            </ListItem>
        </List>
    )
}

export default MenuList
