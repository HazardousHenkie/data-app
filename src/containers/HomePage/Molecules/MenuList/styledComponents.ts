import styled from 'styled-components'

import List from '@material-ui/core/List'

const ListStyled = styled(List)`
    height: 100%;
    display: flex;
    flex-direction: column;
`

export default ListStyled
