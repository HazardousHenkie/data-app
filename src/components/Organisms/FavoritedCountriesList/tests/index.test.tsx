import React from 'react'
import renderer from 'react-test-renderer'

import MockingAppComponent from 'utils/testing/MockingAppComponent'
import MenuList from '../index'

describe('<MenuList />', () => {
    it('should render like snapshot', () => {
        const component = renderer.create(
            <MockingAppComponent>
                <MenuList />
            </MockingAppComponent>
        )

        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
