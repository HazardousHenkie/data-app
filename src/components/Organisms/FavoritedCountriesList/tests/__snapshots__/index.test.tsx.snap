// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuList /> should render like snapshot 1`] = `
<div
  className="sc-AxirZ kNDIFW"
/>
`;
